ui_framework/components/pager/pager.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { KuiPagerButtons } from './pager_buttons';
+
+export function KuiPager({
+    className,
+    startNumber,
+    endNumber,
+    totalItems,
+    hasPreviousPage,
+    hasNextPage,
+    onNextPage,
+    onPreviousPage,
+    ...rest }) {
```

Comment:
I hate to nitpick syntax, but can we indent the closing syntax onto the next line, and add a dangling comma here?

```javascript
  ...rest,
}) {
```

It makes the parameter list feel more enclosed to me, so I can identify it and differentiate it from the body of the function definition more easily. The dangling comma also adheres to [AirBnB's style](https://github.com/airbnb/javascript#commas--dangling).